lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -70,6 +70,11 @@
     private static final CatalogSchemaTableName aliceView = new CatalogSchemaTableName("alice-catalog", "schema", "view");
     private static final Optional<QueryId> queryId = Optional.empty();
 
+    private static final Identity peppa = Identity.forUser("peppa").withGroups(ImmutableSet.of("finance")).build();
```

Comment:
nit: maybe something like `finance_user` and `human_resources_user` would make the tests easier to read without having to refer to how the identity is created?

Reply:
What is the semantic of this user? I don't see it defined in above files.